src/arrays.js

```diff
@@ -17,22 +27,51 @@ const reduce = (elements, cb, memo) => {
   // Combine all elements into a single value going from left to right.
   // Elements will be passed one by one into `cb`.
   // `memo` is the starting value.  If `memo` is undefined then make `elements[0]` the initial value.
+  if (typeof memo === 'undefined') {
+    return elements.reduce((previousTotal, currentValue) => {
+      return cb(previousTotal, currentValue);
+    });
+  }
+  return elements.reduce((previousTotal, currentValue) => {
+    return cb(previousTotal, currentValue);
+  }, memo);
 };
 
 const find = (elements, cb) => {
   // Look through each value in `elements` and pass each element to `cb`.
   // If `cb` returns `true` then return that element.
   // Return `undefined` if no elements pass the truth test.
+  for (let i = 0; i < elements.length; i++) {
+    if (cb(elements[i]) === true) return elements[i];
+  }
+  return 'undefined';
```

Comment:
`undefined` is an actual value.  You don't have to wrap it in quotes and return a string `'undefined'`